config/istio/600-istioegress.yaml

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# Needed by the GCP PubSub Source to communicate with GCP PubSub.
+# Needed by PubSubSource to communicate with Google Cloud Pub/Sub.
 apiVersion: networking.istio.io/v1alpha3
```

Comment:
will this cause problems if Istio mesh is not installed?

Reply:
it is in the istio dir: `config/istio/600-istioegress.yaml`